tests/edit_inputs.sh

```diff
@@ -24,11 +29,24 @@ function edit_ice_in {
 }
 
 function edit_mom_input {
+
+  CHLCLIM="seawifs-clim-1997-2010."$NX_GLB"x"$NY_GLB".v20180328.nc"
```

Comment:
I should probably move this and the FRUNOFF (below) to the individual tests also.

Reply:
I think NX_GLB and NY_GLB need to be defined in individual tests if need to be overridden since their defaults are defined in default_vars. As to CHLCLIM, maybe define it in `fv3_conf/*.in`?

Reply:
I just moved the variables CICEGRID, CICEMASK, CHLCLIM, FRUNOFF to default_vars.sh. I guess individual tests can override any one of these if needed in the future. I don't think it's a good idea to move them to `fv3_conf/*.in`. Running RT's on Hera with these changes now.